src/components/ValidateCode/ValidateCodeModal.tsx

```diff
@@ -53,20 +43,20 @@ function ValidateCodeModal(props) {
                         src={Illustrations.MagicCode}
                     />
                 </View>
-                <Text style={[styles.textHeadline, styles.textXXLarge, styles.textAlignCenter]}>{props.translate('validateCodeModal.title')}</Text>
+                <Text style={[styles.textHeadline, styles.textXXLarge, styles.textAlignCenter]}>{translate('validateCodeModal.title')}</Text>
                 <View style={[styles.mt2, styles.mb2]}>
-                    <Text style={[styles.fontSizeNormal, styles.textAlignCenter]}>
-                        {props.translate('validateCodeModal.description')}
-                        {!lodashGet(props, 'session.authToken', null) && (
+                    <Text style={styles.textAlignCenter}>
+                        {translate('validateCodeModal.description')}
+                        {!session?.authToken && (
                             <>
-                                {props.translate('validateCodeModal.or')} <TextLink onPress={signInHere}>{props.translate('validateCodeModal.signInHere')}</TextLink>
+                                {translate('validateCodeModal.or')} <TextLink onPress={signInHere}>{translate('validateCodeModal.signInHere')}</TextLink>
                             </>
                         )}
-                        {props.shouldShowSignInHere ? '!' : '.'}
+                        .
```

Comment:
What's the reason of this change?

Reply:
`shouldShowSignInHere` wasn't in the old `propTypes` declaration and it seems this prop is not being passed anywhere